pkg/bpf/endpoint.go

```diff
@@ -40,7 +40,7 @@ type EndpointKey struct {
 // GetKeyPtr returns the unsafe pointer to the BPF key
 func (k EndpointKey) GetKeyPtr() unsafe.Pointer { return unsafe.Pointer(&k) }
 
-// GetValuePtr returns the unsafe pointer to the BPF value
+// GetValuePtr returns the unsafe pointer to the BPF KEY <<<< NOTE!
```

Comment:
Please move this commit to a separate PR. It needs improvements but I think fixing the comment like this is not what we want because it was still not clear to me even after this comment ;-)

Let's move it to a separate PR as we won't have to backport this.

Reply:
@raybejjani Next commit fixes the value receiver for lbmap, but I just noted that I did not fix it for EndpointKey on which the wrong comment was on here. So the comment is wrong, and this too returns a pointer to the temporary. Fixing this forces all other methods on EndpointKey to also use pointer receiver, which is correct as well, as what is a point of taking a copy and then returning pointer to it. That fix will likely cascade further than we want for the bug fix, so I'll not touch it for now.